src/api.ts

```diff
@@ -1,10 +1,10 @@
-import * as httpProxy from 'http-proxy';
-import * as Docker from 'dockerode';
-import * as _ from 'lodash';
-import * as portfinder from 'portfinder';
-import * as git from 'nodegit';
-import * as fs from 'fs-extra';
-import * as path from 'path';
+import httpProxy from 'http-proxy';
```

Comment:
I changed how CJS modules are imported into TypeScript by enabling `esModuleInterop` (https://www.typescriptlang.org/tsconfig#esModuleInterop) which is the preferred option.